traffic_ops/app/lib/MojoPlugins/Job.pm

```diff
@@ -70,8 +70,16 @@ sub register {
 				}
 			);
 
-			my $result = $update_server_bit_rs->update( { upd_pending => 1 } );
-			&log( $self, "Set upd_pending = 1 for all applicable caches", "OPER" );
+			my $use_reval_pending = $self->db->resultset('Parameter')->search( { -and => [ 'name' => 'use_reval_pending', 'config_file' => 'global' ] } )->get_column('value')->single;
+
+			if ( defined($use_reval_pending) && $use_reval_pending == 1 ) {
```

Comment:
isn't it 

&& $use_reval_pending == '1'

but i dunno. maybe perl is smart enough to figure it out...